core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -253,6 +253,18 @@ export class RteHelperModalComponent {
         );
         return;
       }
+    } else if (this.componentId === this.COMPONENT_ID_TABS) {
+      // Value[0] corresponds to tab Contents.
+      for (let tab = 0; tab < value[0].length; tab++) {
```

Comment:
This is a number, so it should be tabIndex rather than tab

Reply:
Changes made! Thank you !